GUI/src/components/Layout/index.tsx

```diff
@@ -1,9 +1,7 @@
 import { FC, useState, ReactNode, PropsWithChildren } from 'react';
 import { Outlet } from 'react-router-dom';
 import useStore from '../../store/store';
-import {
-  MainNavigation
-} from '@exirain/header/src';
+import { MainNavigation } from '@buerokratt-ria/menu/src';
```

Comment:
**question:** it seems that this menu isn't exported as a module if you have to import it from `src` directly